### PR TITLE
OPS-0 Changing variable from ARG to ENV

### PR DESCRIPTION
**Pull request by runderwoodcr14**
To overwrite default value from the original image, variable needs to be an env variable not an arg, performed local tests and the value now gets overwritten by the one specified in the Dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ RUN apk add unzip make curl
 
 # TODO: verify GPG
 ARG TERRAGRUNT_VERSION='0.21.7'
-ARG DEFAULT_TERRAFORM_VERSION=0.12.29
+ENV DEFAULT_TERRAFORM_VERSION=0.12.29
```
Comment (cytopia) [inline Dockerfile:7]: We could also keep it an arg (so we can change it easily) and just assign this value to ENV
```suggestion
ARG DEFAULT_TERRAFORM_VERSION=0.12.29

ENV DEFAULT_TERRAFORM_VERSION=$DEFAULT_TERRAFORM_VERSION
```

Comment (ronny-panknin-flaconi) [reply Dockerfile:7]: > We could also keep it an arg (so we can change it easily) and just assign this value to ENV

sounds good

Comment (runderwoodcr14) [reply Dockerfile:7]: Noted and will submit the change

**Review by ronny-panknin-flaconi — APPROVED** [review]

**Review by cytopia — APPROVED** [review]
:+1:

**Pull request merged**